src/stylesheets/components/_closing.scss

```diff
@@ -0,0 +1,48 @@
+.closing {
+  color: $florence-blue;
+  margin: 0px auto 50px;
+  position: relative;
+  width: 80%;
+
+  @media ($screen-md-up) {
+    margin-bottom: 150px;
+    width: 50%;
+  }
+
+  p:last-of-type {
+
+    @media ($screen-md-up) {
+      width: 50%;
+    }
+  }
+
+  .section-intro {
+    font-size: 3em;
+    width: initial;
+  }
+}
+
+.booklet {
+  display: none;
+
+  @media ($screen-md-up) {
+    display: block;
+    height: 160px;
+    position: absolute;
+    right: 0;
+    top: 0;
+    transform: translate(-100px, 250px);
```

Comment:
Absolute positioning the booklet is fine but if the amount of content in the closing were ever to change, the positioning would need to be re-adjusted. I'd recommend floating the last paragraph to the left (with a right margin and maybe width), and then remove all the absolute positioning stuff here and the breakpoint. Then the image will maintain it's normal place in the document flow.